skimage/transform/_warps.py

```diff
@@ -74,6 +74,13 @@ def resize(image, output_shape, order=1, mode=None, cval=0, clip=True,
     preserve_range : bool, optional
         Whether to keep the original range of values. Otherwise, the input
         image is converted according to the conventions of `img_as_float`.
+    anti_aliasing : bool
+        Whether to apply a Gaussian filter to smooth the image prior to
+        down-scaling. It is crucial to filter when resampling the image to
+        avoid aliasing artifacts.
+    anti_aliasing_stddev : {float, tuple of floats}
+        Standard deviation for Gaussian filtering to avoid aliasing artifacts.
+        By default, this value is chosen as 0.25 times the scale factor.
```

Comment:
Is this just based on intuition, or did you find a paper to guide the size of the Gaussian?

Reply:
FreeImage uses the same convention and I checked the results on some examples. I couldn't find any reference that suggests a theoretically proven value. If you have a better suggestion, I'd be happy to incorporate it. If you cannot find a good reference either, I'd appreciate if you could also play around with the parameter and check the results. This is important enough to be checked by more than one person.